src/javascript/JContent/ContentRoute/ToolBar/FileModeSelector/FileModeSelector.jsx

```diff
@@ -48,15 +46,6 @@ export const FileModeSelector = () => {
                     localStorage.setItem(FILE_SELECTOR_GRID_MODE, THUMBNAIL);
                 }
 
-                break;
-            case DETAILED:
-                onChange(GRID);
-                localStorage.setItem(FILE_SELECTOR_MODE, GRID);
-                if (gridMode !== DETAILED) {
-                    onGridMode(DETAILED);
```

Comment:
You can completely get rid of the "gridMode" thing in redux, if we only have one "grid mode" left. You can also remove all JContentConstants.gridMode and directly use LIST/GRID, it will make things much more clear.